lib/cli/package.json

```diff
@@ -36,5 +40,9 @@
     "opencollective": "^1.0.3",
     "shelljs": "^0.7.8",
     "update-notifier": "^2.1.0"
+  },
+  "devDependencies": {
+    "check-node-version": "2.1.0",
+    "npx": "9.6.0"
```

Comment:
Is this used anywhere within our code? I can't find it in this diff.

Reply:
Yes, for running fresh CR(N)A without installing globally (`yarn create` unfortunately does the global install): https://github.com/storybooks/storybook/blob/cac45b496463180d957780d5ca0a142321658626/lib/cli/test/update_fixtures.sh#L14